chemprop/cli/common.py

```diff
@@ -75,8 +75,8 @@ def add_common_args(parser: ArgumentParser) -> ArgumentParser:
     )
     featurization_args.add_argument(
         "--features-path",
-        type=list[str],  # TODO: why is this a list[str] instead of str?
-        help="Path(s) to features to use in FNN (instead of features_generator).",
+        nargs="+",
+        help="Path to features to use in FFN (instead of features_generator). Use as --features-path 0 /path/to/features/for/mol/0.npz 2 /path/to/features/for/mol/2.npz",
```

Comment:
There is only one extra feature vector per input, regardless of the number of components